frontend/stores/notifications/useNotificationsDashboardStore.ts

```diff
@@ -27,40 +27,46 @@ export function useNotificationsDashboardStore() {
     setSearch,
     setStoredQuery,
     isStoredQuery,
-  } = useTableQuery({ limit: 10, sort: 'dashboard:desc' }, 10)
+  } = useTableQuery({ limit: 10, sort: 'timestamp:desc' }, 10)
   const isLoading = ref(false)
 
   async function loadNotificationsDashboards(q: TableQueryParams) {
     isLoading.value = true
     setStoredQuery(q)
-    const result = await fetch<NotifcationDashboardResponse>(
-      API_PATH.NOTIFICATIONS_DASHBOARDS,
-      undefined,
-      undefined,
-      q,
-    )
+    try {
+      const result = await fetch<InternalGetUserNotificationDashboardsResponse>(
+        API_PATH.NOTIFICATIONS_DASHBOARDS,
+        { query: { network: networkId.value } },
+        undefined,
+        q,
+      )
 
-    isLoading.value = false
-    if (!isStoredQuery(q)) {
-      return // in case some query params change while loading
-    }
+      isLoading.value = false
+      if (!isStoredQuery(q)) {
```

Comment:
yes it could be changed while the webservice request is fetched. 

the code at the `await` waits for the ws to return data ... in the meantime the loadNotificationsDashboards could called in the meantime